src/layers/domain/core/device.py

```diff
@@ -46,6 +51,9 @@ class DeviceUpdatedEvent(Event):
     product_team_id: UUID
     ods_code: str
     status: "DeviceStatus"
+    created_on: str
+    updated_on: Optional[str] = None
```

Comment:
Isnt a device updated event always going to have an updated_on date?

Reply:
Yes, good spot